src/python/pants/util/objects_test.py

```diff
@@ -150,7 +150,8 @@ class FallbackToObject:
         get_docstring(FallbackToObject, fallback_to_ancestors=True, ignored_ancestors=[object])
         is None
     )
+    object_docstring = object.__doc__
+    assert object_docstring is not None
     assert (
-        get_docstring(FallbackToObject, fallback_to_ancestors=True, ignored_ancestors=[])
-        == object.__doc__
+        get_docstring(FallbackToObject, fallback_to_ancestors=True, ignored_ancestors=[]) == object_docstring.rstrip()
```

Comment:
On 3.8, there's an extra `\n` in the docstring for `object`. MyPy complained that `__doc__` is Optional, so we need to make MyPy happy.